Modules/_io/_iomodule.c

```diff
@@ -684,15 +684,6 @@ PyInit__io(void)
     state = get_io_state(m);
     state->initialized = 0;
 
-#define ADD_TYPE(type, name) \
```

Comment:
Can you keep the macro but modify it to use PyModule_AddType()?

Reply:
I redfined ADD_TYPE parameter also 
AS-IS: ADD_TYPE(type, name)
TO-BE: ADD_TYPE(type,)